src/index.js

```diff
@@ -9,13 +10,25 @@ var shisha = {
      * @param data
      * @param locals
      * @param callback
+     * @param capath
      */
-    smoke: function (data, locals, callback) {
+    smoke: function (data, locals, callback, capath) {
```

Comment:
Indeed that makes sense, however, I did not wish to break BC at this point. I hope it can be refactored to an options object in a later major release